packages/devtools/src/launcher.js

```diff
@@ -108,7 +108,8 @@ async function launchChrome (capabilities) {
     return browser
 }
 
-function launchBrowser (capabilities, product) {
+function launchBrowser (capabilities, browserType) {
+    const product = browserType === BROWSER_TYPE.firefox ? BROWSER_TYPE.firefox : BROWSER_TYPE.chrome
```

Comment:
puppeteer only supports firefox and chrome.

Reply:
Chromium Edge as well

Reply:
Sorry for not being clear, `product` option support only `chrome` and `firefox`.

Reply:
https://github.com/puppeteer/puppeteer/blob/950ae334ca9c4c37b5a34dc61596a89596fd2bc6/src/node/Launcher.ts#L803

Reply:
I'm able to run tests in both headless and headful modes in the Edge browser, of course.
https://github.com/mgrybyk/webdriverio-devtools/blob/master/config/wdio.EDGE.conf.ts#L12